docs/install/_system_requirements.rst

```diff
@@ -6,20 +6,23 @@ System requirements
 Operating systems
 -----------------
 
- - Windows Vista, 7, 8.1, 10
+ - Windows Vista, 7, 8.1, 10 
  - Debian/Ubuntu: 14.04, 16.04 and up - anything that's *not* end-of-life.
  - Linux: Any system with Python 2.7
  - Mac OSX 10.9, 10.10 and 10.11
  - Debian/Raspberry Pi packages: Wheezy or later
  - *(planned)* Android: 4.2.2 or later
 
 
-.. TODO - Limited support
+Limited support
+^^^^^^^^^^^^^^^
+ - Windows XP cannot be used to install Kolibri server, but could potentially work as a client device if the browsers are `as up-to-date as possible <https://support.mozilla.org/en-US/questions/1173904>`_.
```

Comment:
Ok, let's clear the inaccuracies that we can asap: 

- Do you think Android should go to `Limited support`?
- Which versions of Safari & OSX/macOS do we support?

Reply:
I think we need to clearly distinguish client support and server support. This is my rough understanding...

* Android server - unsupported
* Android client - 4.2+ supported
* Mac server - 10.6+ supported as PEX
* Mac client - all browsers supported
* iOS server - unsupported
* iOS client - safari unsupported, chrome & FF supported
* Windows server - 7, 8, 10 supported
* Windows client - IE 11+, chrome, FF supported

Reply:
Thanks for the suggestion of a simple overview @indirectlylit -- I'm runnning with that, just merging it a bit with the existing list. It will have to be fixed according to our support matrix/table that accounts for Kolibri version, too... but for now, I think this is an improvement that's valid.